### ISSUE TITLE
Embed Block: WordPress Embed Block accessibility issue

### ISSUE THREAD

**Issue by Rishit30G** [WordPress/gutenberg#70153]
### Description

The current “Continue More” text in WordPress Embed block fails to meet WCAG AA and AAA accessibility standards, likely due to insufficient contrast. To improve accessibility, it is recommended to use a darker text color to ensure better contrast against the background, making it easier for all users to read.

### Step-by-step reproduction instructions

- Add a new post 
- Add an Embed Block in the editor 
- Paste a post url in the input field of embed block ( make sure the post has 8-10 lines of content ) 
- Save the post and view it on frontend 
- Notice the 'Continue More' button is barely visible, it can create issue for low vision users 

### Frontend 

![Image](https://github.com/user-attachments/assets/95bb3a11-b1dc-49d7-992f-9b705d0ba3a5)

### Accessibility Issue

<img width="529" alt="Image" src="https://github.com/user-attachments/assets/c1679f00-5e50-434e-a37e-3adb3dda7dec" />

### Environment info

- Gutenberg 20.8.0
- WordPress 6.8.1
- Theme TT5 
- Google Chrome 
- MacOS Sequoia 

### Please confirm that you have searched existing issues in the repo.

- [x] Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

- [x] Yes

### Please confirm which theme type you used for testing.

- [ ] Block
- [ ] Classic
- [ ] Hybrid (e.g. classic with theme.json)
- [ ] Not sure

**Comment by sabernhardt** [comment]
The excerpt and the meta links are also too light. The light colors come from `wp-embed-template.css`, so this belongs on Core Trac.

**Comment by sabernhardt** [comment]
I moved this to [Trac 63460](https://core.trac.wordpress.org/ticket/63460).

**Comment by Rishit30G** [comment]
Thanks @sabernhardt!

**Comment by swissspidy** [comment]
Closing as this is indeed for Trac only
